visual-testing-app/src/index.tsx

```diff
@@ -1,13 +1,35 @@
 import ReactDOM from 'react-dom';
-import { PortalsContainer } from '@commercetools-frontend/application-components';
+import { useState } from 'react';
 import { ThemeProvider } from '@commercetools-uikit/design-system';
+import {
+  PortalsContainer,
+  themesOverrides,
+} from '@commercetools-frontend/application-components';
 import Application from './application';
+import ThemeSwitcher, {
+  ThemeName,
+} from './components/theme-switcher/theme-switcher';
 
-ReactDOM.render(
-  <>
-    <ThemeProvider />
-    <PortalsContainer />
-    <Application />
-  </>,
-  document.getElementById('app')
-);
+const Main = () => {
+  const [selectedTheme, updateTheme] = useState<ThemeName>('default');
+  const isLocalEnvironment = process.env.NODE_ENV === 'development';
+
+  return (
+    <>
+      <ThemeProvider
+        theme={selectedTheme}
+        themeOverrides={themesOverrides[selectedTheme]}
+      />
+      {isLocalEnvironment ? (
+        <ThemeSwitcher
+          selectedTheme={selectedTheme}
+          onThemeChange={(newTheme) => updateTheme(newTheme)}
+        />
+      ) : null}
```

Comment:
We add the theme switcher only in development mode, where it's useful.
_(Otherwise, it would be included in Percy snapshots)_